### PR TITLE
Frontend/feature/word cloud widget

### PR DESCRIPTION
**Pull request by steffenslavetinsky**

### CODE REVIEW

File: src/widgets/WordCloudView/MenuBar.tsx [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+        }
+    }, [stopwords, stopwordsInputValue, onChangeStopwords]);
+
+    useEffect(() => {
+        setStopwordsInputValue(stopwords.join(', '));
+    }, [stopwords]);
+
+    return (
+        <Menu tw="w-[360px]">
+            <Menu.ColumnSelect
```
Comment (steffenslavetinsky) [inline src/widgets/WordCloudView/MenuBar.tsx:92]: Remove and use normal select

File: src/widgets/WordCloudView/Cloud.tsx [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+                                highlightRows(d.rowIds);
+                            })
+                            .on('mouseout', () => highlightRows([]))
+                            .on('click', (e, d) => {
+                                selectRows(d.rowIds);
+                                e.stopPropagation();
+                            })
+                            .attr('text-anchor', 'middle')
+                            .text((d) => d.text || '')
+                            .attr('fill', (d, i) =>
```
Comment (steffenslavetinsky) [inline src/widgets/WordCloudView/Cloud.tsx:160]: change to do on update (to update color on simply filtering)

File: src/widgets/WordCloudView/Cloud.tsx [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+                    ? 1
+                    : 0.3
+            )
+            .attr('fill', (d, i) =>
+                d.rowIds.some((index) => isIndexFiltered[index])
+                    ? categoricalScale(i % categoricalPalette.maxClasses).hex()
+                    : '#000000'
+            )
+            .on('mouseover', (_, d) => {
+                d.rowIds.some((index) => isIndexFiltered[index]) &&
```
Comment (steffenslavetinsky) [inline src/widgets/WordCloudView/Cloud.tsx:222]: always update highlighted

File: src/widgets/WordCloudView/WordCloudView.tsx [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+        if (!rows) {
+            return [{}, 0];
+        }
+
+        const splitter =
+            splitStringsBy.length > 1 ? new RegExp(splitStringsBy) : splitStringsBy;
+
+        let splitRows = rows.map((row) => row.toLowerCase().split(splitter));
+
+        if (columnToCompareBy !== undefined) {
```
Comment (steffenslavetinsky) [inline src/widgets/WordCloudView/WordCloudView.tsx:109]: remove compare by

**Review by neindochoh — APPROVED** [review]

**Pull request merged**